Project.toml

```diff
@@ -27,4 +27,4 @@ Reexport = "0.2, 1.0"
 SciMLBase = "1.90, 2"
 SimpleNonlinearSolve = "0.1, 1"
 SimpleUnPack = "1"
-julia = "1.6"
+julia = "1.9"
```

Comment:
I guess since OrdinaryDiffEq dropped support for < 1.9?

Reply:
yes